src/TraceEvent/TraceLoggingEventId.cs

```diff
@@ -10,19 +11,20 @@ namespace Microsoft.Diagnostics.Tracing
     /// <summary>
     /// TraceLoggingEvnetId is a class that manages assigning event IDs (small 64k numbers)
     /// to TraceLogging Style events (which don't have them).  Because TraceEvent uses EventIDs
-    /// so fundamentally this difficiency is very problematic.   
+    /// so fundamentally this deficiency is very problematic.   
```

Comment:
:bulb: In the future, it helps to place all formatting-only changes and spelling corrections in their own commit, and preferably in their own independent pull request. The primary advantages this offers are:

1. It's easier to expedite spelling corrections so they don't get tied up in code review
2. It makes the code review process less complex for the actual code changes (for example, this pull request would contain a one-line change in one file, and nothing else)

The scope in this pull request is fairly small so I don't see it causing a problem, but it adds up pretty quickly so it's a good thing to keep in mind for the future. 👍